lisa/tools/fdisk.py

```diff
@@ -36,20 +29,18 @@ def make_partition(self, disk_name: str, file_system: FileSystem) -> None:
         # "" => Use default 2048 for 'First sector'
         # "" => Use default 2147483647 as 'Last sector'
         # w => write table to disk and exit
+        mkfs = self.node.tools[Mkfs]
         self.node.execute(
             f"(echo n; echo p; echo 1; echo ; echo; echo ; echo w) | "
             f"{self.command} {disk_name}",
             shell=True,
             sudo=True,
         )
-        if file_system == FileSystem.xfs:
-            mkfs_xfs = self.node.tools[Mkfsxfs]
-            mkfs_xfs.mkfs(f"{disk_name}p1", str(file_system))
-        elif file_system in [FileSystem.ext2, FileSystem.ext3, FileSystem.ext4]:
-            mkfs_ext = self.node.tools[Mkfsext]
-            mkfs_ext.mkfs(f"{disk_name}p1", str(file_system))
+        if "nvme" in disk_name:
```

Comment:
This looks like a hack. what if the name is changed? What's reason to have the name on partition_disk?

Reply:
thanks, I'll fix it by get the info from vm.

Reply:
fixed